undofx/src/main/java/org/fxmisc/undo/UndoManagerFactory.java

```diff
@@ -14,118 +15,249 @@
 
 public interface UndoManagerFactory {
 
-    <C> UndoManager create(
+    /**
+     * Creates an {@link UndoManager} where changes will never be merged and every change emitted from the
+     * changeStream is considered to be a non-identity change.
+     *
+     * @param changeStream the {@link EventStream} that emits changes
+     * @param invert inverts the change
+     * @param apply applies the change (undo/redo)
```

Comment:
Given the names and type signatures of these last three parameters, do these descriptions add any information? I don't think so.